src/lib_c/x86_64-portbld-freebsd/c/unistd.cr

```diff
@@ -8,6 +8,7 @@ lib LibC
   X_OK       = 0x01
   SC_CLK_TCK =    3
 
+  fun chroot(dirname : Char*) : Int
```

Comment:
Please, use consistent naming: `path` vs `dirname`. Latter would be preferred.

Reply:
I have made it `path` when the source/man pages says `path` and `dirname` when the source/man pages says `dirname`. Generally BSD uses `dirname`, linux uses `path'.